.travis.yml

```diff
@@ -3,16 +3,18 @@ group: edge
 sudo: required
 services:
   - mongodb
+  - redis-server
   - docker
 node_js:
-  - '0.10'
   - '4.4'
+  - '6.9'
```

Comment:
I think we should use

- "4.4.3"
- "6.9.1"
Because those are the options in studio and all the other templates have it like this